src/components/content/register/RegisterPanel.tsx

```diff
@@ -3,156 +3,76 @@
-                    <Alert
-                        type={'success'}
-                        message={`Service ${ocl?.name} Registered Successfully`}
-                        closable={true}
-                        onClose={onRemove}
-                        className={'result'}
-                    />
-                );
-            } else {
-                setRegistrationResultDisplay(
-                    <Alert
-                        type={'error'}
-                        closable={true}
-                        showIcon={true}
-                        message={`Service Registration Failed`}
-                        description={registerResult}
-                        onClose={onRemove}
-                        className={'result'}
-                        action={
-                            <Button size='small' type='primary' onClick={onRemove} danger={true}>
-                                Try Again
-                            </Button>
-                        }
-                    />
-                );
-            }
-        }
-    }, [registerResult, ocl?.name, isRegisterSuccessful]);
-
-    useEffect(() => {
-        if (isYamlSyntaxValid) {
-            if (isYamlSyntaxValid && ocl !== null && ocl !== undefined) {
-                const oclTableData = DisplayOclData({ ocl: ocl });
-                if (typeof oclTableData === 'string') {
-                    setIsOclDataValid(false);
-                    setOclDataDisplay(
-                        <Alert
-                            type={'error'}
-                            showIcon={true}
-                            message={`OCL data in the uploaded file not Valid. Error while parsing - ${oclTableData}`}
-                        />
-                    );
-                } else {
-                    setIsOclDataValid(true);
-                    setOclDataDisplay(oclTableData);
                 }
-            }
+            };
         }
-    }, [yamlSyntaxValidationResult, ocl, isYamlSyntaxValid]);
+    }
 
-    const customRequest = () => {
-        if (fileList) {
-            serviceVendorApi
-                .register(ocl as Ocl)
-                .then(() => {
-                    fileList[0].status = 'success';
-                    setRegisterResult('Service Registered Successfully');
-                    setIsRegisterSuccessful(true);
-                })
-                .catch((error: any) => {
-                    setRegisterResult(error.message);
-                    fileList[0].status = 'error';
-                    setIsRegisterSuccessful(false);
-                });
-        }
-        return;
+    const sendRequestRequest = () => {
+        registerService(ocl.current as Ocl, setRegisterRequestStatus, registerResult, files.current[0]);
+        console.log(files.current[0].status);
     };
 
     const setFileData = (file: RcFile): boolean => {
-        setFileList([file]);
+        files.current.push(file);
```

Comment:
Do we need a list here? It seems a bit of confusion as the previous files are not able to be brought up again.

![image](https://user-images.githubusercontent.com/1907997/228117390-136bcd1a-23ee-4f71-962e-fa67ab244095.png)